.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+jobs:
+  publish:
+    docker:
+      - image: cimg/openjdk:8.0-browsers
+    steps:
+      - checkout
+      - run:
+          name: publish
+          command: |
+            set +e
+            echo 'Publishing Voldemort artifacts...'
+            ./gradlew artifactoryPublish
+            echo 'Publishing complete!'
+
+workflows:
+  version: 2
+  publish:
+    jobs:
+      - publish:
+          name: publish
+          context: org-global-cph
+          filters:
+            branches:
+              only: master
```

Comment:
I don't expect us to publish it frequently (I hope we'll do it only once, i.e., go Venice). Therefore, I think restricting it to `master` makes sense.